app/Models/Course.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Course extends Model
+{
+    protected $fillable = ['name', 'is_selected'];
+
+    protected $casts = [
+        'exams' => 'array',
+        'is_selected' => 'boolean',
+    ];
```

Comment:
I would expect a method that implements the HasMany relationship to the course's tests.